pgx-utils/src/sql_entity_graph/pg_extern/mod.rs

```diff
@@ -306,12 +306,12 @@ impl PgExtern {
                     metadata: pgx::utils::sql_entity_graph::metadata::FunctionMetadata::entity(&metadata),
                     fn_args: vec![#(#inputs_iter),*],
                     fn_return: #returns,
-                    schema: None #( .unwrap_or(Some(#schema_iter)) )*,
+                    schema: None #( .unwrap_or_else(|| Some(#schema_iter)) )*,
                     file: file!(),
                     line: line!(),
                     extern_attrs: vec![#extern_attrs],
-                    search_path: None #( .unwrap_or(Some(vec![#search_path])) )*,
-                    operator: None #( .unwrap_or(Some(#operator)) )*,
+                    search_path: None #( .unwrap_or_else(|| Some(vec![#search_path])) )*,
+                    operator: None #( .unwrap_or_else(|| Some(#operator)) )*,
```

Comment:
The reasoning clippy applies here is inappropriate: this is a zero-or-one expansion on a None case. If the macro expands into `.unwrap_or` it will always be attempting to evaluate the inner closure and thus eager evaluation is fully intended.